<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboard</class>
 <widget class="QMainWindow" name="dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="studentLabel_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>Admin Dashboard</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1071</width>
      <height>621</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Students</string>
     </attribute>
     <widget class="QTableView" name="studentTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>1051</width>
        <height>511</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QPushButton" name="printStudents">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="studentLabel_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>Students</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Faculty</string>
     </attribute>
     <widget class="QTableView" name="facultyTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>1051</width>
        <height>511</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QPushButton" name="printFaculty">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="studentLabel_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>Faculty</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="timeTableTab">
     <attribute name="title">
      <string>Time Table</string>
     </attribute>
     <widget class="QPushButton" name="printTimeTablebtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="studentLabel_6">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>Time Table</string>
      </property>
     </widget>
     <widget class="QTableView" name="timeTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>1051</width>
        <height>511</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStudent_2"/>
   <addaction name="actionFaculty_2"/>
   <addaction name="actionActivateUser2"/>
   <addaction name="actionLogout"/>
   <addaction name="actionExit_toolbar"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Data Entry Forms</string>
    </property>
    <addaction name="actionStudent"/>
    <addaction name="actionFaculty"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionActivateUser"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionStudent">
   <property name="text">
    <string>Student</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionFaculty">
   <property name="text">
    <string>Faculty</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionStudent_2">
   <property name="text">
    <string extracomment="Student">Student</string>
   </property>
   <property name="iconText">
    <string extracomment="Student">Student</string>
   </property>
   <property name="toolTip">
    <string>Add Student</string>
   </property>
   <property name="statusTip">
    <string extracomment="Student"/>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFaculty_2">
   <property name="text">
    <string>Faculty</string>
   </property>
   <property name="toolTip">
    <string>Add Faculty Member</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="actionExit_toolbar">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionActivateUser">
   <property name="text">
    <string>Activate User</string>
   </property>
  </action>
  <action name="actionActivateUser2">
   <property name="text">
    <string>Activate User</string>
   </property>
   <property name="toolTip">
    <string>Assign ID, Password</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>dashboard</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit_toolbar</sender>
   <signal>triggered()</signal>
   <receiver>dashboard</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
